apps/client/app/routes/_layout.emails.tsx

```diff
@@ -0,0 +1,114 @@
+import { Form, useLoaderData, useNavigate, useNavigation, useSubmit } from "@remix-run/react";
+import { json, LoaderFunctionArgs } from "@remix-run/node";
+import { useEffect } from "react";
+import { MOCKED_EMAILS } from "../../mocks/emails";
+import { Email } from "../../models/email.model";
+
+
+async function getEmails(searchString: string | null): Promise<Email[]> {
+  await new Promise((resolve) => setTimeout(resolve, 500));
+  if (searchString?.length) {
+    return MOCKED_EMAILS.filter((item) => item.title.toLowerCase().includes(searchString.toLowerCase()));
+  }
+  return MOCKED_EMAILS;
+}
+
+export const loader = async ({
+                               request,
```

Comment:
Could you fix this indentation. 
Setup `prettier/eslint` in this project with this or another ticket could be helpful to automatic fix indentation styles

Reply:
Yeah I thought about implementing prettier and eslint, but I saw that we have a specific issue for it. I will work on it next.